.circleci/config.yml

```diff
@@ -553,6 +559,7 @@ jobs:
           DOCKERIZE_VERSION: v0.6.1
           DJANGO_ALLOW_ASYNC_UNSAFE: 1
           DJANGO_BBB_ENABLED: 1
+          DJANGO_MARKDOWN_ENABLED: 1
```

Comment:
This is useless for e2e, so I may remove it if you want. (e2e tests seems a bit flaky, so it might help removing it)